server/src/main/java/org/elasticsearch/index/query/InnerHitBuilder.java

```diff
@@ -122,17 +130,16 @@
         }, COLLAPSE_FIELD, ObjectParser.ValueType.OBJECT);
     }
     private String name;
-    private boolean ignoreUnmapped;
+    private boolean ignoreUnmapped = DEFAULT_IGNORE_UNAMPPED;
 
-    private int from;
-    private int size = 3;
-    private boolean explain;
-    private boolean version;
-    private boolean seqNoAndPrimaryTerm;
-    private boolean trackScores;
+    private int from = DEFAULT_FROM;
+    private int size = DEFAULT_SIZE;
+    private boolean explain = DEFAULT_EXPLAIN;
+    private boolean version = DEFAULT_VERSION;
+    private boolean seqNoAndPrimaryTerm = DEFAULT_SEQ_NO_AND_PRIMARY_TERM;
+    private boolean trackScores = DEFAULT_TRACK_SCORES;
 
     private StoredFieldsContext storedFieldsContext;
-    private QueryBuilder query = DEFAULT_INNER_HIT_QUERY;
```

Comment:
Is this just unused?

Reply:
Ah, yes, sorry. I was going to leave a comment. Totally unused. At least in this branch